components/eam/src/physics/cam/ndrop.F90

```diff
@@ -1520,6 +1711,44 @@ end subroutine loadaer
 
 !===============================================================================
 
+subroutine get_aer_mmr_sum_1col(imode, nspec, state_q, qcldbrn1d, & !in
+     vaerosolsum, hygrosum)   !inout
+
+  !add these for direct access to mmr (in state_q array), density and hygroscopicity
+  use modal_aero_data,   only: lspectype_amode, specdens_amode, spechygro, lmassptr_amode
```

Comment:
some of these are used multiple times in the module, maybe define them as module variables?

Reply:
Not all of these are used in all subroutines, but yes, maybe at some point they should just be module variables (especially because most of the subroutines are not public).